packages/sdk/test/integration/gc_test.ts

```
@@ -419,32 +731,107 @@
     await client2.activate();
 
     await client1.attach(doc1, { syncMode: SyncMode.Manual });
+    assert.equal(
+      versionVectorHelper(doc1.getVersionVector(), [
+        { actor: client1.getID()!, lamport: Long.fromNumber(1) },
+      ]),
+      true,
+    );
     doc1.update((root) => (root.point = { x: 0, y: 0 }));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)